scripts/cmake/vcpkg_download_distfile.cmake

```
@@ -265,8 +265,9 @@ If you do not know the SHA512, add it as 'SHA512 0' and re-run this command.")
             WORKING_DIRECTORY "${DOWNLOADS}"
         )
 
+        message("${output}")
```

Would it be better instead to just not redirect output in the first place? (That is, remove `OUTPUT_VARIABLE` and `ERROR_VARIABLE` above entirely rather than reprinting it here?)

Should we get rid of the `message` call on line 239 now?

---

@BillyONeal  Thanks for the suggestion.

I think line 239 is still necessary and we can enclose the output like this, which could be better.

```suggestion
        if((NOT vcpkg_download_distfile_QUIET) OR (NOT "${error_code}" EQUAL "0"))
           message("${output}")
        endif()
```

After such a change, the log output is displayed when `vcpkg_download_distfile_QUIET` is off or error code is not zero.

---

>After such a change, the log output is displayed when vcpkg_download_distfile_QUIET is off or error code is not zero.

Isn't the whole point of this change that we *always* want to display the output, even when the exit code is zero?